docker-compose.local.yml

```diff
@@ -1,20 +1,33 @@
 version: '3'
 services:
+  joplindb:
+    image: postgres
+    ports:
+     - "5432:5432"
+    environment:
+      POSTGRES_DB: 'joplin'
+      POSTGRES_USER: 'joplin'
+      POSTGRES_PASSWORD: '39h3zd3j9Pc57mH3QsiS6sb4'
```

Comment:
Is this ok to have captured in Git?

Reply:
That should be fine in my opinion since it is for local use only and does not affect any PR, staging or production deployments. It may be possible to set up a no-password requirement; not sure how long this might take.

Reply:
it might make sense to change this password to reflect the fact that it's for local use and insecure, something like `POSTGRES_PASSWORD: insecureLocalOnlyPassword#123` instead of having it look like a somewhat secure random string of numbers chillin in the codebase

Reply:
It looks like it works fine without any password set up, tested it repeatedly via redeployment and with two separate clients. Will commit change to branch.